javascript-modal/main.js

```diff
@@ -0,0 +1,15 @@
+var $openModalBtn = document.querySelector('.open-modal-btn');
+var $modal = document.querySelector('.modal');
+var $no = document.querySelector('.no-btn');
+
+function modalOn(event) {
+  $modal.style.display = 'block';
+}
+
+function modalOff(event) {
+  $modal.style.display = 'none';
+}
```

Comment:
This will end up being inline styling (aka the `style` attribute on elements), and that is bad news, because it will mess up specificity in your stylesheet to override this. Instead, you should make a class with the styling `display: none` that you add and remove as necessary